README.md

```diff
@@ -20,10 +20,19 @@ Here are some of the benefits of using this library:
 
 **API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
 
+### Packages
+
+| Namespace                              | Package                                                                                                                              |

```

Comment:
Maybe `Project` is better?

Reply:
Probably doesn't need to be a table actually. Just a list, the package name and whatever we put for namespace/project will be the same.

Reply:
@jobullen do you mind making these changes?